src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotificationsTrait/config.jelly

```diff
@@ -3,4 +3,7 @@
   <f:entry title="${%Communicate Unstable builds to Bitbucket as Successful}" field="sendSuccessNotificationForUnstableBuild">
     <f:checkbox/>
   </f:entry>
+  <f:entry title="${%Don't Communicate not build jobs to Bitbucket}" field="disableNotificationForNotBuildJobs">
```

Comment:
I'm not a native speaker, but this should actually be "not built", not "not build". Unfortunately this is everywhere now.